lib/l10n/app_en.arb

```diff
@@ -1474,7 +1474,7 @@
   "@volumeNormalizationModeSelectorSubtitle": {
     "description": "Subtitle for the dropdown that selects the replay gain mode"
   },
-  "volumeNormalizationModeSelectorDescription": "Hybrid (Track + Album):\nTrack gain is used for regular playback, but if an album is playing (either because it''s the main playback queue source, or because it was added to the queue at some point), the album gain is used instead.\n\nTrack-based:\nTrack gain is always used, regardless of whether an album is playing or not.\n\nAlbums Only:\nVolume Normalization is only applied while playing albums (using the album gain), but not for individual tracks.",
+  "volumeNormalizationModeSelectorDescription": "Hybrid (Track + Album):\nWorks like track-based, but when playing multiple tracks from the same album in a row, works like album-based for them instead.\n\nTrack-based:\nTrack gain is always used, regardless of whether an album is playing or not.\n\nAlbums Only:\nVolume Normalization is only applied while playing albums (using the album gain), but not for individual tracks.\n\nAlbum-based:\nAlbum gain is always used, but if it is not available (e.g. Jellyfin does not provide it for singles), track gain is used.",
```

Comment:
This is incorrect. "Hybrid" will not always use the tracks gain. When playing an album, the album gain will be used, that's why it's called hybrid. For albums album gain, for anything else track gain.  
And now with your improvement it's album gain for albums *and* successive tracks from the same album, and track gain for anything else.

And let's rename this to `Dynamic` instead of `Hybrid (Track + Album)`